lightning/src/ln/channel.rs

```diff
@@ -7436,6 +7436,12 @@ impl<SP: Deref> OutboundV1Channel<SP> where SP::Target: SignerProvider {
 		Ok(self.get_open_channel(chain_hash))
 	}
 
+	/// Returns true if we can resume the channel by sending the [`msgs::OpenChannel`] again.
+	pub fn is_resumable(&self) -> bool {
+		!self.context.have_received_message() &&
+			self.context.cur_holder_commitment_transaction_number == INITIAL_COMMITMENT_NUMBER
```

Comment:
Is this method necessary? It seems like `have_received_message()` should cover it, I don't see how we could be beyond the initial commitment number if we've only just sent our init 🤔 I'm ok with it because it makes the callsite more readable, though.